src/styles/utils/_mixins.scss

```diff
@@ -1,6 +1,16 @@
-@mixin hover($_property, $_toValue) {
-  transition: #{$_property} 0.3s;
-  &:hover {
-    #{$_property}: $_toValue;
-  }
+@mixin small_img_block($small_img_size) {
+  width: $small_img_size;
+  height: $small_img_size;
+}
+
+@mixin flex_row_between {
```

Comment:
will be better to create extends mixin is overkill

Reply:
done, left only one, should i delete it also?

Reply:
For my mind, not should, but will be good